src/plugins/ovp/components/video-screen/index.js

```diff
@@ -36,6 +40,30 @@ function VideoScreen(props) {
   const canDelete = policy.isGranted(`${qname}:delete`);
   const canUpdate = policy.isGranted(`${qname}:update`);
 
+  const schema = node.schema();
+  const hasMedialiveChannel = schema.hasMixin('triniti:ovp.medialive:mixin:has-channel') && node.has('medialive_channel_arn');
+  const medialiveChannelArn = node.get('medialive_channel_arn');
+  const medialiveRequest = useResolver('triniti:ovp:request:search-videos-request', hasMedialiveChannel && medialiveChannelArn ? {
+    channel: `video-medialive-${nodeRef}`,
+    initialData: {
+      count: 1,
+      page: 1,
+      q: `medialive_channel_arn:"${medialiveChannelArn}"`,
+      derefs: ['medialive_channel_state'],
+    },
+  } : null);
```

Comment:
@meghedi isn't this included with the get-node-request for the video? iirc it is included by the `HasChannelEnricher`